cds-snc/cppd-medical-report
# Styling conditions screen and Delete condition

# Description

Style the `conditions` screen and enable deletes

# Test Instructions

- Create some conditions
- Observe the new conditions screen
- Try deleting one!
